storage/tests/c/flash.h

```diff
@@ -24,7 +24,9 @@
 #include <stdlib.h>
 #include "secbool.h"
 
-#define FLASH_BYTE_ACCESS 1
+#ifndef FLASH_QUADWORD
```

Comment:
I think `FLASH_BIT_ACCESS` and `FLASH_QUADWORD` (which BTW is equivalent to `FLASH_BLOCK_WORDS==4`) should be treated as independent and should both be specified in the build flags rather than being inferred from one another.

Reply:
fixed https://github.com/trezor/trezor-firmware/commit/31ca162522180b08acda83e0f00f8e615f1e8999